opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/ipfix.adoc

```diff
@@ -48,12 +47,31 @@ The IPFIX TCP Parser accepts packets received by a <<telemetryd-listener-tcp, TC
 
 This parser does not currently have any configurable parameters.
 
+===== Configure IPFIX Listener on a Minion
+
+To enable and configure a _TCP Listener_ for IPFIX on Minion, connect to the _Karaf Console_ and set the following properties:
+
+[source]
+----
+$ ssh -p 8201 admin@localhost
+...
+admin@minion()> config:edit --alias tcp-5000 --factory org.opennms.features.telemetry.listeners
+admin@minion()> config:property-set name IPFIX
+admin@minion()> config:property-set class-name org.opennms.netmgt.telemetry.listeners.TcpListener
+admin@minion()> config:property-set parameters.port 50000
+admin@minion()> config:property-set parsers.0.name IPFIX 
+admin@minion()> config:property-set parsers.0.class-name org.opennms.netmgt.telemetry.protocols.common.parser.ForwardParser
```

Comment:
```suggestion
admin@minion()> config:property-set parsers.0.class-name org.opennms.netmgt.telemetry.protocols.netflow.parser.IpfixTcpParser
```

Reply:
to clarify, shouldn't "netflow" be "ipfix"

Reply:
No, it all sits in the `netflow` package. Ipfix is netflow v10 but renamed as netflow is a Cisco product and ipfix is an open standard.

Reply:
Ok. Thanks. Updated!